UPGRADE.md

```diff
@@ -1,3 +1,15 @@
+## MINOR BC BREAK: Doctrine\DBAL\Query\QueryBuilder
```

Comment:
What version do you think this will be in? I figured I'd just leave this out for now.